mindsdb/libs/controllers/transaction.py

```diff
@@ -135,6 +135,7 @@ def _execute_learn(self):
             self._call_phase_module('ModelTrainer')
 
             self._call_phase_module('ModelAnalyzer')
+            self.persistent_model_metadata.update()
```

Comment:
This should be inside the ModelAnalyzer

Reply:
Why are the other .update() calls not inside the phases ?
Either way, will move it